hack/update-generated-completions.sh

```diff
@@ -4,9 +4,6 @@
 
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-# ensure we have the latest compiled binaries
-"${OS_ROOT}/hack/build-go.sh"
```

Comment:
this should be basically free if we've already done a build anyway.  and if you haven't done a build, the completions you generate are going to be stale/wrong, no?